canonical/ubuntu.com
# WD-8332 Dev - Rebrand /containers/what-are-containers

## Done
-  Rebranded /containers/what-are-containers
- The work was been picked from here https://github.com/canonical/ubuntu.com/pull/13900 
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please find the design here https://www.figma.com/design/vHVDziBUEWpxKdYG2lLIZN/What-are-containers?node-id=1-3755&t=u9BkaiaziFcyI5Ly-0
- [Demo](https://ubuntu-com-13931.demos.haus/containers/what-are-containers)

## Issue / Card

Fixes # [WD-8332](https://warthogs.atlassian.net/browse/WD-8332)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-8332]: https://warthogs.atlassian.net/browse/WD-8332?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
